app-drawer-layout/README.md

```diff
@@ -52,6 +52,27 @@ With an app-header-layout:
 </app-drawer-layout>
 ```
 
+Add the `drawer-toggle` attribute to toggle the drawer on tap events:
```

Comment:
"Add the `drawer-toggle` attribute to elements inside `app-drawer-layout` that toggle the drawer on tap events:"